dev-docs/bidders/smaato.md

```diff
@@ -13,20 +13,31 @@ pbjs: true
 pbs: true
 pbs_app_supported: true
 prebid_member: true
+getFloor: true
 ---
 
 ### Table of Contents
 
+- [Registration](#smaato-registration)
+- [Note](#smaato-note)
 - [Bid Params](#smaato-bid-params)
 - [App Object](#smaato-app-object)
 - [Example Ad Units](#smaato-example-ad-units)
 - [First Party Data](#smaato-first-party)
 - [Test Parameters](#smaato-test-parameters)
 
+<a name="smaato-registration" />
+
 ### Registration
 
 The Smaato adapter requires setup and approval from the Smaato team, even for existing Smaato publishers. Please reach out to your account team or prebid@smaato.com for more information.
 
+<a name="smaato-note" />
+
+### Note
+
+The Smaato adapter supports bidfloor with 'USD' currency.
```

Comment:
You don't need to document this -- your adapter code is converting the floor values into the CURRENCY you choose, so this statement may confuse pubs.

Reply:
We request USD from getFloor and don't use a floor when result of that is not USD.
We see similar notes in other docs and want to be specific.

Reply:
> We see similar notes in other docs

I would have said something to them had I noticed. :-)

How about "The Smaato adapter will convert bidfloors to 'USD' currency as needed."

Reply:
thank you for explaining the reasoning :)

i've updated this line